OutaTime/OutaTime/ViewControllers/DatePickerViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  DatePickerViewController.swift
+//  OutaTime
+//
+//  Created by Cora Jacobson on 6/8/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import UIKit
+
+protocol DatePickerDelegate {
+    func destinationDateWasChosen(_ date: Date)
+}
+
+class DatePickerViewController: UIViewController {
+
+    @IBOutlet weak var destinationDatePicker: UIDatePicker!
+    
+    var delegate: DatePickerDelegate?
```

Comment:
Don't forget to make delegates weak, and have the protocol constrained to AnyObject (I think you had this project done before we talked about this in standup, not meaning to nag, just reminding)